azure-pipelines.yml

```diff
@@ -408,7 +413,7 @@ jobs:
       REDUCE_SDK: 'True'
       AZURE_CLI_TEST_TARGET_PROFILE: '2019-03-01'
 
-- job: RunAutomationReduced20180301
+- job: RunAutomation20180301
```

Comment:
Why do you run reduced tests for profile 2019-03-01 but 2018-03-01 not? What is the difference?

Reply:
This is the current test settings for python 3.6. I modified the name to reflect the settings.

Reply:
I see. Just out of curiosity, ReducedAutomation is run when REDUCE_SDK is set to true.  But through the [scripts](https://github.com/Azure/azure-cli/blob/dev/scripts/ci/test_automation.sh#L35), I cannot see anything special besides reduced modules.

Reply:
Not familiar with testing details, add @haroldrandom for digging in.